md/product-versioning.md

```diff
@@ -3,13 +3,24 @@ Discover how Bonita manages its product and artifacts versions and how it can im
 
 ## Product Versions
 
-Bonita follows the usual semantic versioning X.Y.z:
+Bonita follows calendar versioning: YYYY.n:
+* YYYY year of the release
+* n is the release number. Two versions are released every year.
+
+### Technical Id
+
+A technical Id also exists and it follows the usual semantic versioning X.Y.z:
 * X is the major version number. It changes when non-backward compatible changes are implemented.
 * Y is the minor version number. It version changes when backward compatible features are added. A minor version is released every six months to improve continuously Bonita.
 * Z is the maintenance version number. The maintenance versions contain fixes. Maintenance versions are released every month between two minor versions for the current minor version. These versions are cross-compatible by default for the same minor version.
 
 For example, upgrading from a 7.12.2 to a 7.12.5 will not require any migration. However, migrating from a 7.11.5 to a 7.12.2  or to  8.0.1 will require a migration.
```

Comment:
You can also add than since 2021.1, Bonita Studio supports projects that are created in any maintenance version for a given minor version.